python_modules/libraries/dagster-fivetran/dagster_fivetran/asset_defs.py

```diff
@@ -127,7 +131,9 @@ def _build_fivetran_assets(
                 connector_id=connection_metadata.connector_id,
                 name=connection_metadata.name,
                 connector_url=connection_metadata.connector_url,
-                schema_config=connection_metadata.schemas,
+                schema_config=FivetranSchemaConfig.from_schema_config_details(
```

Comment:
[1] `FivetranConnectorTableProps` was updated to take a `FivetranSchemaConfig` object instead of a `Mapping[str, Any]` one. The goal here is to keep `FivetranConnectorTableProps` as close as possible to the new implementation.